api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,27 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceLogsSent = 1;
+  uint64 numDeviceBundlesSent = 2;
+  uint64 numAppLogsSent = 3;
+  uint64 numAppBundlesSent = 4;
+  uint64 num4xxResponses = 5;
+  google.protobuf.Timestamp lastDeviceLogSendTime = 6;
+  google.protobuf.Timestamp lastAppLogsSendTime = 7;
+
+  bool isLogProcessingDeferred = 8;
+  uint64 numTimesDeferred = 9;
+  google.protobuf.Timestamp lastLogDeferTime = 10;
+
+  uint64 avgBundleLogCount = 11;
+  uint64 avgBundleProtoByteCount = 12;
```

Comment:
Average over what time period?? Is it just LogsSent/BundlesSent?

Reply:
just LogsSent/BundlesSent

Reply:
Doesn't seem useful to send something which can trivially be computed. Adds no info.

Reply:
Makes sense. I'll remove the avgBundleLogCount.
But, currently we do not send total bundle bytes sent. Should I remove avgBundleProtoByteCount also and add TotalBundleProtoBytesSent?

Reply:
We have a metric for the total bytes send to the log API. Maybe that is sufficient. 
If not I think we should have a complete set of byte counts (in addition to the complete set of event counts).

Reply:
Added separate proto byte count fields for Device events and App events.